src/assets.ts

```diff
@@ -272,6 +301,34 @@ export function createWebLaunchConfiguration(programPath: string, workingDirecto
 }`;
 }
 
+export function createBlazorWebAssemblyLaunchConfiguration(workingDirectory: string): string {
+    return `
+{
+    "name": "Launch Blazor web assembly project in Chrome",
+    "type": "pwa-chrome",
```

Comment:
Is `pwa-chrome` a type built into VSCode or does it depend on another extension being installed?

Reply:
It's built into the node debugger for VSCode (not yet released though)